### PR TITLE
5844 dispute market card

### PR DESCRIPTION
**Pull request by bthaile**
added partial dispute market card, other tasks fill in the outcomes. Other PR for changing `challenge` to `dispute` verbiage.

### CODE REVIEW

**Comment by sharkcrayon** [comment]
(I'm reviewing from a visual standpoint, @stephensprinkle please still review the code)

@bthaile Is there something I can do to get markets in the Dispute state so I can see markets listed here? (I know we were discussing this in #UI, just bringing the discussion over here.)

Also, two adjustments:
- Can you update the Null state text to match the text shown in the screenshot in CH? You can also see this in the Reporting_1228 page in the Sketch file (latest is uploaded to Google Drive).
- Can you update the Null state styles to match the styles shown in the null state screenshot in CH? It is ok to do this in the shared component and effect everything throughout the site.

File: src/modules/reporting/components/dispute-market-card/dispute-market-card.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      <article className={CommonStyles.MarketCommon__container}>
+        <div className={CommonStyles.MarketCommon__topcontent}>
+          <div className={CommonStyles.MarketCommon__header}>
+            <ul className={BasicStyles.MarketBasics__tags}>
+              {p.market.tags.length > 1 &&
+                <li>Tags</li>
+              }
+              {(p.market.tags || []).map((tag, i) => i !== 0 &&
+                <li key={p.market.id + tag}>
+                  <button onClick={() => toggleTag(tag, { pathname: makePath(MARKETS) }, p.history)}>
```
Comment (stephensprinkle) [inline src/modules/reporting/components/dispute-market-card/dispute-market-card.jsx]: @sharkcrayon — are we going to have tag filtering in the reporting views?

Comment (stephensprinkle) [inline src/modules/reporting/components/dispute-market-card/dispute-market-card.jsx]: @sharkcrayon — are we going to have tag filtering in the reporting views?

File: src/modules/reporting/selectors/select-dispute-markets.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import store from 'src/store'
+
+export default function () {
+  return selectMarketsInDispute(store.getState())
+}
+
+export const selectMarketsInDispute = createSelector(
+  selectMarkets,
+  (markets) => {
+    if (!markets || markets.length === 0) {
```
Comment (stephensprinkle) [inline src/modules/reporting/selectors/select-dispute-markets.js]: You can use `lodash` `isEmpty` here — I’ve been progressively moving that direction in course to clean up these kinds of conditionals.

**Comment by bthaile** [comment]
@sharkcrayon I'll grab the newest mocks and check out the styling:
Two ways to get disputed markets to populate:
1. change the code in src/modules/reporting/selectors/select-dispute-markets.js to filter on PRE_REPORTING markets.
or if you don't want to mod the code you can update the database
2.
in augur-node dir
>sqlite3 augur.db
>update market_state set reportingState="CROWDSOURCING_DISPUTE" where reportingstate="PRE_REPORTING";

that'll change regular canned markets to crowd sourcing dispute state. 

>update market_state set reportingState="PRE_REPORTING" where reportingstate="CROWDSOURCING_DISPUTE";

to change them back.

File: src/modules/reporting/components/dispute-market-card/dispute-market-card.jsx (hunk 1 above)
Comment (sharkcrayon) [reply src/modules/reporting/components/dispute-market-card/dispute-market-card.jsx]: I have markets showing up now, and I see that clicking on a tag on a market on the Dispute view brings users to the filtered Markets view.

That's fine. We'll adjust after MVP.

**Comment by sharkcrayon** [comment]
@bthaile Getting close! One more thing: The market cards need the "Dispute Round" text and number in the top right of the card.

**Comment by bthaile** [comment]
Ok, I’ll figure out addimg the dispute round. It would be movk data, i think its not availabe currently in middleware or at least I’ve not seen it available

**Pull request merged**